### PR TITLE
Feature/apiview blob backup

### PR DESCRIPTION
**Pull request by praveenkuttappan**
Pipeline to copy blob from production apiview storage account to staging apiview storage account

### CODE REVIEW

File: .azure-pipelines/apiview-sync-staging.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+stages:
+  - stage: 'Main'
+    jobs:
+      - job: 'BlobCopy'
+        pool:
+          vmImage: 'vs2017-win2016'
+
+        steps:
+        - pwsh: |
+            Invoke-WebRequest -MaximumRetryCount 10 -Uri "https://aka.ms/downloadazcopy-v10-windows" -OutFile "azcopy.zip" | Wait-Process;
```
Comment (scbedd) [inline .azure-pipelines/apiview-sync-staging.yml:18]: I don't see azcopy locally in the repo anymore, so I don't have an issue with this.

File: .azure-pipelines/apiview-sync-staging.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
+parameters:
```
Comment (weshaggard) [inline .azure-pipelines/apiview-sync-staging.yml:1]: We should try to eventually move all these pipeline files under eng\pipelines.

cc @chidozieononiwu

File: .azure-pipelines/apiview-sync-staging.yml (hunk 1 above)
Comment (weshaggard) [reply .azure-pipelines/apiview-sync-staging.yml:18]: Is there a reason we have to use azcopy? Can we not use https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/deploy/azure-file-copy?view=azure-devops?

File: .azure-pipelines/apiview-sync-staging.yml [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  - originals
+  - codefiles
+
+
+stages:
+  - stage: 'Main'
+    jobs:
+      - job: 'BlobCopy'
+        pool:
+          vmImage: 'vs2017-win2016'
```
Comment (weshaggard) [inline .azure-pipelines/apiview-sync-staging.yml:14]: We should be using the `windows-2019" which is the latest image.

File: .azure-pipelines/apiview-sync-staging.yml (hunk 1 above)
Comment (praveenkuttappan) [reply .azure-pipelines/apiview-sync-staging.yml:18]: @weshaggard : Sorry for the delayed response here. Reason for using azcopy is to use "sync" operation supported by azcopy  which attempts to copy only what is missing in destination and runs whole task faster. I didn't see that option using azure file copy task. Daily sync job doesn't have to go over al blobs in source this way.

File: .azure-pipelines/apiview-sync-staging.yml (hunk 3 above)
Comment (praveenkuttappan) [reply .azure-pipelines/apiview-sync-staging.yml:14]: I will update it in my next Pr.

File: .azure-pipelines/apiview-sync-staging.yml (hunk 2 above)
Comment (praveenkuttappan) [reply .azure-pipelines/apiview-sync-staging.yml:1]: I will submit a new PR to make this change.

**Pull request merged**